gaj23/bigbirds
# Gaj/feature/seen list

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. Include screenshots as needed.

This includes features that render a top 30 list, a user's seen list, and then details about a particular bird. 

This closes issue #4 #5 #8 #9 

### Problems Encountered & Resulting Solutions  
> State any problems encountered & include screenshots as needed. 
> Describe your code changes in detail for reviewers.

Some challenges were encountered when attempting to pass data down. This was due to my unfamiliarity using react hooks. 

I've implemented the use of useLocation to pass down information.

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

Both `TopList` and `PersonalList` render a list of birds (`Bird` & `Sighting` respectively). Each bird can be clicked on so that we can view details of a particular bird (see `Details`). 


### How should this be tested?
> What steps need to be taken to confirm functionality? 

Please clone `git@github.com:gaj23/bigbirds.git` locally, ensure that npm is installed, and put `npm start` into the CLI. Please click between `Top 30`, `Home`, and `Seen` to confirm that the information renders. Then, please click into a detailed view using the arrows on the renders lists. 

### Steps To Take
> Any questions and possible steps moving forward.

Although I've been able to pass information down, my next biggest challenge will be to pass information __between__ two components that share the same data. I'm questioning if that's the best practice.

#### Checklist:
